client/src/subxt.rs

```diff
@@ -16,7 +16,7 @@ pub trait Identity: System {
 
     type Mask: Parameter + Member + Default + From<[u8; 32]> + Into<[u8; 32]>;
 
-    type Gen: Parameter + Member + Copy + Default + CheckedAdd + From<u8> + Ord;
+    type Gen: Parameter + Member + Copy + Default + CheckedAdd + From<u16> + Into<u16> + Ord;
```

Comment:
`From<u16>` implies `Into<u16>`
```suggestion
    type Gen: Parameter + Member + Copy + Default + CheckedAdd + From<u16> + Ord;
```

Reply:
Are you sure?

Reply:
From<u16> means u16 implements Into<T>